flink-streaming-connectors/flink-connector-kafka/src/main/java/org/apache/flink/streaming/connectors/kafka/FlinkKafkaConsumer.java

```diff
@@ -632,6 +632,7 @@ public void run() {
 						//  ------------  commit current offsets ----------------
 
 						// create copy of current offsets
+						//noinspection unchecked
```

Comment:
I think you can remove the line after the annotation. I don't think we have a policy about this, but I've seen it in many places.